docs/development/setup.md

```diff
@@ -30,6 +30,7 @@ bin/setup
 ```shell
 bin/dev
 # OR
+[yarn install --frozen-lockfile]
```

Comment:
Why the square brackets around this?

Reply:
just to indicate optional as you may have already run this. 

For some reason bin/setup was failing and `yarn build` etc did not work until i did this. If you have already run yarn install i do not think it matters. 

I will change to below  if clearer?

```shell
yarn install --frozen-lockfile
# THEN
bin/dev
# OR
yarn build && yarn build:css
...
```

Reply:
Looks clearer to me :+1: 
Happy for further discussion if other devs disagree! 🙂